src/ui/ui_manager.py

```diff
@@ -269,6 +269,10 @@ def _keep_item(self, item_finder: ItemFinder, img: np.ndarray, do_logging: bool
             if ("potion" in x.name) or (self._config.items[x.name].pickit_type == 0): continue
             include_props = self._config.items[x.name].include
             exclude_props = self._config.items[x.name].exclude
+            #Disable include params for uniq, rare, magical if ident is disabled in params.ini
+            #if (not self._config.char["id_items"]) and ("uniq" in x.name or "magic" in x.name or "rare" in x.name or "set" in x.name):
+            if (not self._config.char["id_items"]) and any(item_type in x.name for item_type in ["uniq", "magic", "rare", "set"]):
+                include_props = False 
             if not (include_props or exclude_props):
```

Comment:
Some people prefer not to identify items. If they have `id_items=0` shouldn't we set include *and* exclude props to none? Some properties like sockets won't show up on unidentified items iirc.